src/CSnakes.Runtime/CPython/Import.cs

```diff
@@ -18,6 +18,27 @@ internal static PyObject Import(string name)
         return PyObject.Create(module);
     }
 
+    internal static PyObject Import(string name, string code, string path)
+    {
+        
+        nint codeObject = Py_CompileString(code, path, InputType.Py_file_input);
+        if (codeObject == IntPtr.Zero)
+        {
+            throw PyObject.ThrowPythonExceptionAsClrException();
+        }
+
+        nint pyName = AsPyUnicodeObject(name);
+        nint pyCode = AsPyUnicodeObject(code);
+        nint pyPath = AsPyUnicodeObject(path);
```

Comment:
Why not use [`PyObject.From(string)`](https://github.com/tonybaloney/CSnakes/blob/35db15a98c5a4b604b900fabb9325909277b103e/src/CSnakes.Runtime/Python/PyObject.cs#L598-L607) for these and where the `Py_DecRefRaw` calls below could also be handled through `using` declarations and disposal?